lib/screens/news_screen.dart

```diff
@@ -19,10 +24,36 @@ class NewsScreen extends StatelessWidget {
                 "Blop I'm the subtitle",
                 style: Theme.of(context).textTheme.bodyText1,
               ),
+              Obx(
+                () => Column(
+                  mainAxisAlignment: MainAxisAlignment.center,
+                  children: articleController.articles
+                      .map(
+                        (Article a) => Row(
```

Comment:
Widget
But it's not your job to do that

Reply:
Yes it's just a functional example for the one that will develop this page